R/splitByRanks.R

```diff
@@ -19,10 +19,12 @@
 #'   will be dropped. This setting can be tweaked by defining
 #'   \code{empty.fields} to your needs. (default: \code{na.rm = TRUE})
 #'
-#' @param keep_reducedDims \code{TRUE} or \code{FALSE}: Should the
+#' @param keep.reduceddims \code{TRUE} or \code{FALSE}: Should the
```

Comment:
Capital D was removed so that the naming is harmonized. We have only lowercase characters in parameters (except *.FUN which is established way to denote function)

Reply:
Scater and scuttle use  `dimred` very widely. This could be keep.dimred